gunicorn/glogging.py

```diff
@@ -238,6 +239,21 @@ def setup(self, cfg):
                     TypeError
             ) as exc:
                 raise RuntimeError(str(exc))
+        elif cfg.logconfig_json:
+            config = CONFIG_DEFAULTS.copy()
+            if os.path.exists(cfg.logconfig_json):
+                try:
+                    config_json = json.load(open(cfg.logconfig_json))
+                    config.update(config_json)
+                    dictConfig(config)
+                except(
+                    json.JSONDecodeError,
+                    AttributeError,
+                    ImportError,
+                    ValueError,
+                    TypeError
+                ) as exc:
+                    raise RuntimeError(str(exc))
```

Comment:
Would you like to submit a separate PR that changes this code to `raise RuntimeError(...) from None` after we merge this? There are a few more instances of this pattern, so it would be nice to leave it to a separate PR.

Reply:
I can certainly do that.